lib/src/main/java/space/taran/arklib/domain/index/ResourceIndex.kt

```diff
@@ -1,20 +1,22 @@
 package space.taran.arklib.domain.index
 
 import space.taran.arklib.ResourceId
+
+import kotlinx.coroutines.flow.Flow
 import java.nio.file.Path
 
 interface ResourceIndex {
 
-    suspend fun allIds(): Set<ResourceId> = allIds(null)
+    val updates: Flow<ResourceUpdates>
```

Comment:
Is it important if we use `Flow` or `SharedFlow` here?

Reply:
No, `Flow` is ok